### ISSUE TITLE
Users want map and filters to stay the same when going back from a profile

### ISSUE THREAD

**Issue by chriswhong** [NYCPlanning/labs-zap-search#339]

**Comment by chriswhong** [comment]
When navigating from filter view, to project view, and back again, the filter view starts fresh with its query params.  

- The map loads again, triggering an animation to its default view, when the user may have had it zoomed into an area of interest
- The list resets to the first 30 projects that match the filter params, even if the user had added  more to the list by clicking "Load More" (this can be very frustrating if user has loaded hundreds of applications and just wants to quickly view details about one and continue down the list)

In short, the list view should persist.  This is possible by making the project view a child route that renders on top of the parent.

**Comment by hannahkates** [comment]
This could be addressed by #431 instead, which would be a much simpler, more flexible fix.

**Comment by andycochran** [comment]
I'd close #431 in favor of this approach. Tho this is more complicated to implement, it's the better practice from a UX perspective.
